standards/using-3rd-party-open-source-products.md

```diff
@@ -0,0 +1,32 @@
+# Using 3rd-party open source products
+
+In order to make effective commercial decisions, it is important to recognise the characteristics of open source software that we find desirable. Many organisations ‘open’ their code, but monetise the software through means that restrict the freedom of the user. Some open source software does not have a strong community, and can be risky to use should problems arise which the MOJ does not understand how to resolve.
```

Comment:
Might be worth saying something about why we *do* want to use open source products, too?

Reply:
Strongly agree (about explaining that open source = good). Something along the lines of:
"Neverthless, there are many significant benefits to using open source offerings, including:
- Avoidance of vendor or proprietary 'lock-in'.
- Flexibility of customisation.
- Faster and more transparent patching and security fixes.
- Highly cost effective.
- Common convergence on standards.
- Transparent record of development."

Reply:
I wonder, actually, if we need to go into that level of detail. Considering open source is [part of the technology code of practice](https://www.gov.uk/government/publications/technology-code-of-practice/technology-code-of-practice#make-things-open)… Though it's not very strong on the why (though it used to be, I thought?).

Reply:
Perhaps we don't need that level of detail - but if we don't, who will? Put another way, does it hurt to put some concrete, 'measurable' benefit?

Reply:
If other sources (especially on GOV.UK) do the work for us, I'd prefer to just link to them. But if we can't find those, then we should definitely put detail here.

Reply:
Works for me :-)